src/postal/smtp.clj

```diff
@@ -26,15 +26,26 @@
         [postal.support :only [make-props]])
   (:import [javax.mail Transport Session]))
 
+(defn- send-single [transport ^javax.mail.Message jmsg]
+  (.sendMessage transport jmsg (.getAllRecipients jmsg))
+  {:code (.getLastReturnCode transport)
+   :error :SUCCESS
+   :message (.getLastServerResponse transport)})
+
+(defn- send-multiple [transport jmsgs]
+  (doseq [^javax.mail.Message jmsg jmsgs]
+    (send-single transport jmsg))
+  {:code 0 :error :SUCCESS :message "messages sent"})
```

Comment:
Also, this single return value will mask all the return values from the `send-single` invocations, which could have an error. Likely want to return a vec here that the user can scan through.

Reply:
I believe [sendMessage](https://docs.oracle.com/javaee/7/api/javax/mail/Transport.html#sendMessage-javax.mail.Message-javax.mail.Address:A-) will throw an exception if any of the messages fail. For that reason, the API for sending multiple messages is problematic. (If something fails, it's hard for the caller to figure out which messages succeeded.) But it's fine for callers that don't care to track that anyway.

I tried to not change the interface at all, since you might have users that depend on it. I'm not sure what policy you have for breaking changes.